### ISSUE TITLE
Screen Reader: Privacy Policy: The logo link of 'TRUSTe' is announced without part of displayed text

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76946]
## Action Performed:
Prerequisite:
- User is logged in

1. Using JAWS + Chrome, open the URL: https://www.expensify.com/privacy
2. Press the G key to focus on the "TRUSTe" logo link
3. Observe the screen reader announcement

## Expected Result:
The 'TRUSTe' logo should announce its whole displayed text, e.g.:
- 'TRUSTe Verified Privacy, Powered by TrustArc'

## Actual Result:
When a screen reader user navigates to the 'TRUSTe' logo link, it does not announce the complete visible text associated with it, the user can hear only:
- 'TRUSTe'

HTML:
<a href="//privacy.trustarc.com/privacy-seal/validation?rid=c36c3001-c900-4fd9-a736-67a4cb4c4602" target="_blank"><img style="border: none" src="//hostedseal.trustarc.com/privacy-seal/seal?rid=c36c3001-c900-4fd9-a736-67a4cb4c4602" alt="TRUSTe"></a>

## Area issue was found in:
Privacy Policy

## Failed WCAG checkpoints
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User impact:
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested resolution:
Update the alt text to announce the whole displayed text, e.g.:
- 'TRUSTe Verified Privacy, Powered by TrustArc'
https://www.w3.org/WAI/tutorials/images/informative/

## Environment(s)
Windows 11 Home - Chrome

## Screenshots/Videos
<img width="1918" height="967" alt="Image" src="https://github.com/user-attachments/assets/f3c5597d-72d9-4131-a484-3a3ff0fb8350" />

https://github.com/user-attachments/assets/ce6eab9a-41b6-4a1b-a7f7-2c3b1589b883

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6999333
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
not app related

**Comment by rushatgabhane** [comment]
fixed
